planning/planning_validator/autoware_planning_latency_validator/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.14)
+project(autoware_planning_latency_validator)
```

Comment:
[nits] Same as other module, following name maybe proper in order to align naming convention.

Example:

- autoware_planning_validator_latency_validation
- autoware_planning_validator_latency_check
- autoware_planning_validator_latency_checker :star: 
- autoware_planning_validator_latency_monitor